Libraries/Components/Keyboard/Keyboard.js

```diff
@@ -155,12 +158,12 @@ Keyboard = KeyboardEventEmitter;
 Keyboard.dismiss = dismissKeyboard;
 Keyboard.scheduleLayoutAnimation = function(event: KeyboardEvent) {
   const {duration, easing} = event;
-  if (duration) {
+  if (duration != null) {
     LayoutAnimation.configureNext({
       duration: duration,
       update: {
         duration: duration,
-        type: (easing && LayoutAnimation.Types[easing]) || 'keyboard',
+        type: (easing != null && LayoutAnimation.Types[easing]) || 'keyboard',
```

Comment:
:+1:

Reply:
In fact, the `easing` type at the top of this file could probably be even stronger by using `Type` from `LayoutAnimation.js` instead of any string.